pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -97,8 +104,8 @@ func NewPodController(
 		nodeName:              nodeName,
 		podCache:              podCache,
 		interfaceConfigurator: interfaceConfigurator,
+		vfDeviceIDUsageMap:    make(map[string][]podContainerDeviceIDInfo),
```

Comment:
I think we need a RWMutex to protect the map, or use sync.map?

Reply:
added a RWMutex to protect the read and wrote segment.